README.md

```diff
@@ -16,7 +16,8 @@ A dbt profile can be configured to run against Spark using the following configu
 | host    | The hostname to connect to                         | Required                | `yourorg.sparkhost.com`  |
 | port    | The port to connect to the host on                 | Optional (default: 443) | `443`                    |
 | token   | The token to use for authenticating to the cluster | Required                | `abc123`                 |
-| cluster | The name of the cluster to connect to              | Required                | `01234-23423-coffeetime` |
```

Comment:
Did you intend to remove this? If so, do you want tio remove it from the example below (line 34) and the credentials schema?

Reply:
Nope - that was a fat finger in vim i think...